example/spec/parallel/discoverGranules/DiscoverGranulesWithExecutionNamePrefixSpec.js

```diff
@@ -94,8 +101,10 @@ describe('The DiscoverGranules workflow', () => {
     beforeAllCompleted = true;
   });
 
-  afterAll(() =>
-    Promise.all([
+  afterAll(async () => {
+    await waitForCompletedExecution(workflowExecution.executionArn);
+    await granulesApiTestUtils.deleteGranule({ prefix: stackName, granuleId: 'MOD09GQ.A2016358.h13v04.006.2016360104606' });
+    await Promise.all([
```

Comment:
I commented on #2252 as well, I think we need to consider a Promise.allSettled or pMap with stop on failure set to false for a bunch of these, so as much is cleaned up as possible, even if there's a failure.     Obviously out of scope for this PR, but raising for visibility/future discussion.

Reply:
[`815243d` (#2254)](https://github.com/nasa/cumulus/pull/2254/commits/815243d4a145724b531e1380b1140e63f738bb96)

Reply:
Right, pAll is what I meant.   💯